.github/workflows/e2e_aws.yaml

```diff
@@ -214,7 +215,7 @@ jobs:
       - name: Force AWS clean-up
         if: failure() && steps.runTests.outcome == 'failure'
         env:
-          res_basename: "ci-caa-${{ env.GITHUB_RUN_ID }}_${{ env.GITHUB_RUN_ATTEMPT }}"
+          res_basename: "${{ env.RESOURCES_BASENAME }}"
```

Comment:
Here you are actually changing what you introduced in previous commit. Could you squash the commits?